.cspell.json

```diff
@@ -1,40 +1,44 @@
 {
-    "version": "0.1",
-    "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/master/cspell.schema.json",
-    "language": "en",
-    "words": [
-        "bitauth",
-        "bitjson",
-        "bytecode",
-        "cimg",
-        "circleci",
-        "codecov",
-        "commitlint",
-        "dependabot",
-        "editorconfig",
-        "esnext",
-        "execa",
-        "exponentiate",
-        "globby",
-        "ifps",
-        "ipfs",
-        "Keccak",
-        "libauth",
-        "mkdir",
-        "prettierignore",
-        "sandboxed",
-        "submodules",
-        "transpiled",
-        "Typechain",
-        "typedoc",
-        "untracked"
-    ],
-    "flagWords": [],
-    "ignorePaths": [
-        "package.json",
-        "package-lock.json",
-        "yarn.lock",
-        "tsconfig.json",
-        "node_modules/**"
-    ]
-}
\ No newline at end of file
+  "version": "0.2",
```

Comment:
I think this is more personal dev preferences and we should not commit this file in the repo.

Same for these extensions:

```
    "eamodio.gitlens",
    "streetsidesoftware.code-spell-checker",
```